source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -9,287 +9,472 @@ The global.conf file contains the global configuration settings for Sponge. This
-                                                                in global config. Config doesn't need to be enabled either,
-                                                                because it will always fall back to the world config.
-max-chunk-unloads-per-tick                integer   100         The maximum number of queued unloaded chunks that will be
-                                                                unloaded in a single tick.
-mob-spawn-range                           integer   4           Specifies the radius (in chunks) of where creatures
-                                                                will spawn. This value is capped to the current
-                                                                view distance setting in server.properties.
-**Portal Agents**                                               A list of all detected portal agents used in this
-                                                                world. In order to override, change the target world
-                                                                name to any other valid world. If world is not found,
-                                                                it will fallback to default.
-"minecraft:default_nether"                world     DIM-1       The default nether world.
-"minecraft:default_the_end"               world     DIM1        The default end world.
-pvp-enabled                               boolean   true        If the would allows PVP combat.
-view-distance                             integer   -1          Allows you to set view distance per world, valid range 
-                                                                3-32, when set to -1 the server-wide view distance will 
-                                                                be used instead
-weather-ice-and-snow                      boolean   true        Enable to allow the natural formation of ice and
-                                                                snow.
-weather-thunder                           boolean   true        Enable to initiate thunderstorms.
-world-enabled                             boolean   true        Enable if this world should be registered.
-========================================  ========  ==========  ===============================================
+block-capturing
+===============
+
+auto-populate
+    If 'true', newly discovered blocks will be added to this config with a default value.
+mods
+    Per-mod block id mappings for controlling capturing behavior
+block-tick-capturing
+    If 'true', individual capturing (i.e. skip bulk capturing) for scheduled ticks for
+    a block type will be performed.
+enabled
+    If 'false', all specific rules for this mod will be ignored.
+block-tick-capturing
+    If 'true', individual capturing (i.e. skip bulk capturing) for scheduled ticks for
+    a block type will be performed.
+enabled
```

Comment:
Something looks wrong here: `enabled` exists twice

Reply:
Because one is under `extrautils2` and the other is under `minecraft`. Since they're the same they can be merged.